x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/jira/translations.ts

```diff
@@ -130,7 +130,7 @@ export const DESCRIPTION_REQUIRED = i18n.translate(
 export const TITLE_REQUIRED = i18n.translate(
   'xpack.triggersActionsUI.components.builtinActionTypes.jira.requiredTitleTextField',
```

Comment:
Should these reflect the changes below? `defaultMessage: 'Summary is required.',`

```suggestion
export const SUMMARY_REQUIRED = i18n.translate(
  'xpack.triggersActionsUI.components.builtinActionTypes.jira.requiredSummaryTextField',
```